Python/codecs.c

```diff
@@ -16,6 +16,12 @@ Copyright (c) Corporation for National Research Initiatives.
 #include "pycore_pystate.h"       // _PyInterpreterState_GET()
 #include "pycore_ucnhash.h"       // _PyUnicode_Name_CAPI
 
+static const char *codecs_builtin_error_policies[] = {
```

Comment:
why "policies"? Usually, the Python doc refers to that as "error handler".

Reply:
Err, I thought that the handler was for only for the function itself so I wanted to use a different word (here the "policy" is in the design pattern sense and a policy can be implemented by different error handlers). I can change it back to a less formal terminology!

Reply:
I'll just use error handler everywhere. It'll align with the rest of the docs.